relación-con-empresas.md

```diff
@@ -0,0 +1,19 @@
+# Relación de VigoTech Alliance coas empresas
+
+VigoTech Alliance é un meta grupo ou un agregador de grupos cos obxectivos definidos no seu [Manifesto](manifiesto.md)
+
+## Versión corta
+
+VigoTech Alliance non ten relación de forma directa coas empresas, a relación coas empresas é de xeito indirecto a través de cada un dos grupos.
+
+## Versión longa
+
+Cada uns dos grupos de membros de VigoTech Alliance poderá ter as relación con empresas que estimen oportunas sempre e cando respeten os principios de membresia en VigoTech, por exemplo: Patrocinio de meetups e eventos, publicación de ofertas de emprego das empresas, ofertas especificas das empresas os membros dos grupo, etc.
+
+Estas relacións dos grupos con empresas non son necesariamente compartidas polo resto dos grupos membros nin VigoTech Alliance.
+
+O patrocinio de empresas a cada un dos grupos (sexa por separado ou a varios de eles) non supón un ningun caso o patrocinio a VigoTech Allicance.
+
+A aparición do logo de VigoTech Alliance en carteis, folletos, banners, web, etc. xunto o de empresas non supón relación de patrocinio ou colaboración con esas empresas.
```

Comment:
eu terminaría con"ou co organismo que publique o logo, que deberá preguntar antes de utilizalo."

Reply:
Engadino, pero isto eu o someteria a máis opinions ou a mellor redacción, xa que o final podemos aparece se alguen simplemente lista comunidades de galicia

Reply:
Se pasa, non creo que sexa un problema pero por defecto creo que pedir que se confirme antes é razonable. Se alguén non lee o documento e o fai, non pasa nada.. o peor que pode pasar é que pidamos que o quite